src/commands/transaction/construct_transaction/add_action_1/add_action/use_global_contract/mod.rs

```diff
@@ -0,0 +1,121 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = super::super::super::ConstructTransactionContext)]
+pub struct UseGlobalContractAction {
+    #[interactive_clap(subcommand)]
+    mode: UseGlobalActionMode,
+}
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = super::super::super::ConstructTransactionContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+#[non_exhaustive]
+/// Choose a global contract deploy mode:
+pub enum UseGlobalActionMode {
+    #[strum_discriminants(strum(
+        message = "as-global-hash       - Deploy code as a global contract code hash (immutable)"
```

Comment:
looks like `strum_discriminant` has been copy-pasted from `src/commands/transaction/construct_transaction/add_action_1/add_action/deploy_global_contract/mod.rs` . 
The discriminant should be `use-global-hash` and help message adjusted accordingly. 

Also worth checking the same in #456